templates/spin_sidebar_v2.hbs

```diff
@@ -251,17 +251,21 @@
                         Kubernetes
                 </label>
                 <ul class="menu-list accordion-menu-item-content">
-                        <li><a {{#if (active_project request.spin-full-url "/spin/v2/kubernetes" )}} class="active"
-                                        {{/if}} href="{{site.info.base_url}}/spin/v2/kubernetes">Spin on Kubernetes</a>
+                        <li><a {{#if (active_content request.spin-path-info "/kubernetes" )}} class="active" {{/if}}
```

Comment:
Hi @karthik2804 
My apologies; I don't understand the changes here `active_content` vs `active_project` and `path-info` vs `full-url`.
When I preview the documentation side by side with a checkout of this PR they seem to have the same behaviour. I feel like I am totally missing something here.

Reply:
HI @tpmccallum This address the issue pointed out in this [PR comment](https://github.com/fermyon/developer/pull/1070#issuecomment-1840843426)